Meta/run.sh

```diff
@@ -86,14 +86,19 @@ if [ "$installed_major_version" -lt "$SERENITY_QEMU_MIN_REQ_VERSION" ]; then
     die
 fi
 
+SERENITY_SPICE_SERVER="spicevmc"
+if [ -z "$SERENITY_SPICE" ] && "${SERENITY_QEMU_BIN}" -chardev help | grep -iq qemu-vdagent; then
+    SERENITY_SPICE_SERVER="qemu-vdagent,clipboard=on,mouse=off"
+fi
+
 SERENITY_SCREENS="${SERENITY_SCREENS:-1}"
-if (uname -a | grep -iq WSL) || (uname -a | grep -iq microsoft); then
+if  [ "$SERENITY_SPICE" ]; then
+    SERENITY_QEMU_DISPLAY_BACKEND="${SERENITY_QEMU_DISPLAY_BACKEND:-spice-app}"
+elif (uname -a | grep -iq WSL) || (uname -a | grep -iq microsoft); then
     # QEMU for windows does not like gl=on, so detect if we are building in wsl, and if so, disable it
     # Also, when using the GTK backend we run into this problem: https://github.com/SerenityOS/serenity/issues/7657
     SERENITY_QEMU_DISPLAY_BACKEND="${SERENITY_QEMU_DISPLAY_BACKEND:-sdl,gl=off}"
-elif [ $SERENITY_SCREENS -gt 1 ] && "${SERENITY_QEMU_BIN}" --display help | grep -iq sdl; then
-    SERENITY_QEMU_DISPLAY_BACKEND="${SERENITY_QEMU_DISPLAY_BACKEND:-sdl,gl=off}"
-elif ("${SERENITY_QEMU_BIN}" --display help | grep -iq sdl) && (ldconfig -p | grep -iq virglrenderer); then
+elif [ $SERENITY_SCREENS -gt 1 ] && "${SERENITY_QEMU_BIN}" --display help && (ldconfig -p | grep -iq virglrenderer); then
```

Comment:
I'm not sure what your intention is here.

Reply:
Trying again to get the qemu display thing figured out :P qemu-vdagent doesn't work on SDL (currently -- I filed a bug report so it might be fixed on the next release) so I tried to make it only preferred on windows and when there are multiple screens. (I have clearly made a typo though :/ )

I see you made #8692 about it, so I'll just comment there and take this change out.